lib/vault/defaults.rb

```diff
@@ -15,6 +15,17 @@ module Defaults
     # @return [String]
     SSL_CIPHERS = "TLSv1.2:!aNULL:!eNULL".freeze
 
+    # The default number of attempts.
+    # @return [Fixnum]
+    RETRY_ATTEMPTS = 2
+
+    # The default backoff interval.
+    # @return [Fixnum]
+    RETRY_BASE = 0.05
+
+    # The default amount of time for a single request to timeout.
+    RETRY_MAX_WAIT = 2.0
```

Comment:
This comment should get updated to match the description of the field from the readme, which is nice and clear